### ISSUE TITLE
Update Roll-up Task BUG

### ISSUE THREAD

**Issue by wmpp** [kairosdb/kairosdb#445]
Version 1.2.0 
Create a Roll-up Task：
![image](https://user-images.githubusercontent.com/27722236/37196928-a325423e-23b3-11e8-8d8a-a538c6317294.png)
LOG:
03-09|16:02:28.590 [qtp663466110-120] INFO  [RollUpManager.java:82] - Scheduling rollup test_rollup03-09|16:02:28.594 [qtp663466110-120] INFO  [RollUpManager.java:86] - Roll-up task RollUpJob.def85bc5-3409-491a-9999-782447d05b72-test_rollup scheduled. Next execution time Fri Mar 09 16:12:28 CST 2018
![image](https://user-images.githubusercontent.com/27722236/37197013-f7db4af8-23b3-11e8-9905-0ea3fd459bda.png)

But when updating this rollup by ID, ID changed.and New ID cannot be used for queries, updates, etc. old ID can be.
LOG:
03-09|16:13:15.824 [qtp663466110-133] INFO  [RollUpManager.java:108] - Updating schedule for rollup test_rollup
03-09|16:13:15.824 [qtp663466110-133] INFO  [RollUpManager.java:112] - Roll-up task RollUpJob.def85bc5-3409-491a-9999-782447d05b72-test_rollup scheduled. Next execution time Fri Mar 09 16:23:15 CST 2018
![image](https://user-images.githubusercontent.com/27722236/37197240-f6de28b8-23b4-11e8-8385-7089ffcb1de2.png)
Use new ID query. response:java.io.FileNotFoundException: http://**:8089/api/v1/rollups/6423f71b-9c2f-45eb-bece-8ab5592c30ec

Use new ID update. response:{"errors":["Resource not found for id 6423f71b-9c2f-45eb-bece-8ab5592c30ec"]}

use old ID query.
![image](https://user-images.githubusercontent.com/27722236/37197813-20626864-23b7-11e8-9166-ce96d14c11f3.png)

Using the old ID update can also. BUG？？？

**Comment by jsabin** [comment]
Thank you for your detailed bug. 

I have just finished changing rollups to use Cassandra as the data store. This allows multiple Kairos nodes to split up the rollups and execute them. I have verified that the bug you reported has been fixed in this version. The fixed version is checked into the develop branch and we hope to be releasing it soon with some other kairos fixes being made. 

I tested this by updating a rollup task using both the UI and the rest API. Both worked for me.

Please try out this version and let me know if it works for you.

**Comment by tkozybski** [comment]
@jsabin  Not sure if you are referring to 1.2.1 but the bug still exists in that version.

**Comment by tkozybski** [comment]
I don't remember what I did that caused it. I used a bit of both. My guess is that it happened when I changed the interval to per minute and then changed it back (and the rollup task were running in the same time).

**Comment by tkozybski** [comment]
It happened again. It's not happening because of ongoing rollup, this was not the case this time.

I created a rollup, updated it via WizTools RESTClient using PUT method. The ID got changed. When im querying for /api/v1/rollups it returns the new id, but this ID can't be used for anything, responses says resource not found. But when I query using the original ID it works fine.

Log from kairosdb.log
> 04-14|13:43:14.686 [Roll-up-Modification-Checker-0] INFO  [SchedulingManager.java:200] - Scheduled roll-up task XXX's with id RollUpJob.2811ef59-5b87-4e72-ac81-d8dfaf6ef43c. Next execution time Sat Apr 14 14:43:14 CEST 2018
04-14|13:53:25.826 [Roll-up-Modification-Checker-0] INFO  [SchedulingManager.java:226] - Updating schedule for rollup XXX's
04-14|13:53:25.827 [Roll-up-Modification-Checker-0] INFO  [SchedulingManager.java:230] - Roll-up task XXX's with id RollUpJob.faffb78c-807d-4255-af2f-e8866b50269d scheduled. Next execution time Sat Apr 14 14:53:25 CEST 2018

Attaching screenshoots from RESTClient with the responses. Bottom one is using the original ID.

![beztytulu](https://user-images.githubusercontent.com/5121875/38768013-fb099e78-3fec-11e8-8d60-fd6de02e32d5.jpg)
